access_id: REPLACEME
access_secret: REPLACEME
liftName: dummylift1
connectionURL: ip-someip
baseURL: somewebsite.com
auth_server_url: https://dev.kone.com/api/v2/oauth2/token
requestHeaders:
  Content-Type: "application/x-www-form-urlencoded"
token_req_payload:
  grant_type: "client_credentials"
  scope: "robotcall/*"
timeout: 60 # Seconds
